app/views/kitten/components/form/select.html.erb

```diff
@@ -48,3 +48,15 @@
         ],
   }) %>
 <% end %>
+
+<%= example "select with label" do %>
+  <label class="k-Select--label">Please select a language:</label>
```

Comment:
Est-ce qu'on peut relier le label à l'input avec l'attribut `for` ? Il faut regarder si on peut passer la prop `id` au composant.